src/compiler/diagnosticMessages.json

```diff
@@ -2989,6 +2989,14 @@
         "category": "Message",
         "code": 6148
     },
+    "Did you mean to import the default export, with no curly braces":{
+        "category": "Message",
+        "code": 6149
+    },
+    "Did you mean to import the named import '{0}' by using curly braces":{

```

Comment:
And:

`Module '{0}' has no exported member '{1}'. Did you mean to use 'import {1} from "{0}"'instead?`